src/governance.spec.ts

```diff
@@ -1,22 +1,28 @@
 import { AnonymousIdentity } from "@dfinity/agent";
 import { mock } from "jest-mock-extended";
-import { ICP, LedgerCanister, Vote } from ".";
+import { GovernanceError, ICP, LedgerCanister, Vote } from ".";
```

Comment:
I'd rather specify the file it's importing, than using `"."` it could lead to circular references.

Reply:
Webstorm auto refactor import function 😫. Fixed thanks.